src/openrct2/core/Numerics.hpp

```diff
@@ -30,6 +30,18 @@ namespace Numerics
         return (((_UIntType)(x) << shift) | ((_UIntType)(x) >> (limits::digits - shift)));
     }
 
+    /**
+     * Bitwise left rotate of lowest 4 bits
+     * @param x unsigned 8-bit integer value
+     * @param shift positions to shift
+     * @return rotated value
+     */
+    static constexpr uint8_t rol4(uint8_t x, size_t shift)
```

Comment:
Ugly, but to silence a CI warning:
```suggestion
    [[maybe_unused]] static constexpr uint8_t rol4(uint8_t x, size_t shift)
```
cf. https://travis-ci.org/OpenRCT2/OpenRCT2/jobs/540565137